frontend/public/static/img/bookmarks.svg

```diff
@@ -0,0 +1 @@
QS7vFb2n766UGtfPvwRfU0vb5lxaMRQxDZtC5fcygAle0BhNb2jfOVDq/QX1BLe1FaEs7CLKUFYipda54sKJyjLoU7PoVS1u7LHWMGj2ZKm3Wv7Qx6tIq/FSiu/7StlCX9jz90v57psKuX7G0rr/Uihrd9TOoM1pto5Z4+zS7fsUSJy+ldf9XReRuVwNfUI95fs6qayJyU2s0g1nezjIjdUmu9GN+s96O0mGJVX5M6r20pR/zW2rlKm13Sh2Tlx7Un2gNZrLUb6CxqMcsS1/wX2og5r6rdWq9KupRz89ZtdQxamldf6+KWtIz/UsN6lL/u8YSV1G7LgAwaqlj1I8BzPE69rmo935tBrO0xf4l3OC3RUQeYbnfRGNQH2jcDOpj3lY2XV7IDX5t3tEaGKJuumwGtaR9/kVW04a3tQaGqAVjM6glbZ8ufQmopIoa3fWXtH269CWgd7UGJSltlt+09CWgkiqqajOoJd2SsvSKWtIY9YbWYDOoR7ytbFr0CaXVPUxL/2ZKRV0zLnkytdR9/qaSqmonTqaWjUFdKXUytbQb/Nq8pzUoRalbqDl0+8By3i0wtvtag82gllJhFz2RatjTGhgRvYVayl5/DuNTgIv+B0rd68/lceQP0XGvfUlK6eo3ZXEbiYjsY/mHZyZRakVd2p2oXfa0BiUodYyaS9cPMKgAyu36WVEzU2pQWVEzsxnUUp7pV+86WpCSnp1qtRnUEp7pv77wZ6U25VT9R1Ni15/bX3wumxNDqG80LzGoOU2k6nOp6jNFmTuuNdgMagm3pORWUYEyqmqnzaDueFvZsqir0APl+M2VVIld/9Je1xMil9NeoynxmF+OFZVd/8Y/l3BwOqc11FoWh2gGuKY1KKGCbsqx61/6HQRDPdIaMKh5yLEXSKrE01NZraOucDK18c8lnEfNMag5/jcnVVrXf1dE1OtjFugygIdaI8tKC2qO49P6kRTLVVX9JtwMqno4IHM5rqHWLC9RqWcZNoOqHg7IXJYVdcVyUFWldf3qa2IWrOglKgY1H5aXqDpfgQ6U9yhKzkHNeXzdSURua21KexQl56BarahqNQW2g2r9xmn1vfALZnWMGvT82mZQrd/hz4q6PLe0BgAnUzmxujwVdAlcaXv9OQf1YwTs4GRInUgB20G1/rKJbIO62ka1OE6Nm0w556xPpICMg7pi8ZGUuKDC/kQKAK5qDRbOYkXt1fVbdktE1BdvLdzSXzTcx77WAFgPqvVdqRxe16OxWFGDerlmUK3vSgX9gSycxYoapBlU689LqY/kZqDYG1OaQbW+hsqKmrGSJlMWKiqDWgALFdXiNmr0Oqr1KydzX+yHiNyHvbBGr6Nav3LSyukji7tTqpK6/v/UGmSiyJl/SUF1WoNMWL8wzasZVItHyGr3RCRo0J4Ba11/9Al/C8s3bSz93qyMtWvRQbX2B9BkKagW9/tVB0EVkY8RuFSQIUuPGnMyBeBH3lb5sxRUS78XIHAjZjOo/+5tlT9LExBLvxegx3lUADjnbZW/t7QGuRCRmwicgFiyGdSL3lb5e1NrkJnixqmlBDX7ff4Nxc38N4N6ydsqf5aWpwAGFU97W+XvptYgM+V2/c65owAudLTNmbWKavk+f69mRbUaUsDGE6hN1sbcqmZQrS5N3TV0IKWW8/WZm4Kefm4G9Xxrq7xZq6aArYoadJ9ECV1/0BZdZixtowb15CV0/RaDmvNriDY9qzUA2PXnylJF/aLWACijolqaeNQsBTVok6mEimouqCLyEHa6/zPOOfXVpiVMpqz8hW6y9Pt6TmtQQkW1uotjqftXD0OVMEa1VHmaLB1MUXvzEiqquTHqiqWeQl2iYkXNl6WuP6yiOufOwu7rJS1VniZLQQ2eTFmtpoCtsVyTpd+Xeg66DurZzlb5urN6GM4iSxU1eIxq9W5Uq+NTgEE1xdqlt02WHkf5gtbAelAtVZ1Nln5vp5xznedS66Bafb2kpQnHGhG5A1sPLXZW1Tqop7saZcza9TebLFXVzlNU1oNq+SpNwNZkseigWppw+FjaHmbXb5ilh/w6d6esB5UVNR9FV1Trr2S0FNTOZ6csb6HeN7x9WrMU1KCu3+I6qqW/xDbFjVEtBtXSX2IbS8tTx5xzz7R9WAf1sbYGGSuholoKKtBRVS3v9bOi5qd15r/rnDvW9mHmLF7ls8nSFirQMfPfBXCq7cPMWbu81+cygEdao4x8qe2DXQDqLRWZMh9UEdmHrWfCOoN6vO3DzJkP6oqlo4zPt32wC7u7UqUE1dI2cecY1epk6rrWwAhLXX/n8pTFxX7A5r2oPpYq6nHnnHeplJOp/FkaowItB6g5mcqftSdtvduou7C5KwUwqLnyPuO/C5v7/AAnU7nyXpjGnan8Wauo3utPrXb9+wUcmq5ZO5jivYhiF8Djvg8yV0o1hYjcAnBHa5eR1q7fYlBLODnVZKmqtgbV4oJ/KROpmqWgeu/qZVBtsDTzbw2qxa7f2mvPNZYeu3nK95NWt1BL2eevWTrp31pRLa6j3tMaGGNpv/+I72CK1WN+t7QGxlgKKuDZRrV6KOWG1sAYS10/4HnlpNUx6gOtgTGWZv2A56jfLmy+CM3STk0Ia13/1qK/1XXU0oJqrev3jlEtKm3Wby2o3jGqRUXN+kXkIWyFtZiKel9rYJCloG49jmI1qKWcRW2yNPP3TqYsKrGiWjrp712esqiYg9MNlirqSefc2tb+Lmwu5ZQYVGtrqWsTql3YOnRbK+ES303Wgrq2RLULe79BwNYMOJS13/NWUC2NbWoWf08aawXn6eY/7AJ4s6Vhrq6ISImzfmtBXTtAvQvg9ZaGuXpPa2CUta5/K6g/hK0J1dtaA6M+AbCvNcrIelBXFxj8YUvjHP2X1sAiEXkEW2PzE81/2AUAEfk+gD/3Ns+PtaFMDEvj1LXnpg52pkTkTwD8JoC/Q74L5lcB/KvWyLAPtQa5Otr8BxF5DcBrzrkdVK9SeQ7VMsE
```

Comment:
Are there not feather icon alternatives? Seems like these icons are bigger than they should be but if this is all we can get it'll work ok.

Reply:
This is an empty state svg, I got it from the design team, so check in with them?

Reply:
@cacabo, it looks like these svgs are not actually icons, but images. This seems fine to me!